go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/ipfs/go-unixfsnode
 go 1.15
 
 require (
-	github.com/ipld/go-codec-dagpb v1.0.1
-	github.com/ipld/go-ipld-prime v0.7.0
+	github.com/ipld/go-codec-dagpb v1.1.0
```

Comment:
might as well release the version with the two last changes we added, and use it here?